internal/flightplan/decoder_test.go

```diff
@@ -96,9 +212,9 @@ func testCreateWireWorkspace(t *testing.T, opts ...testCreateWireWorkspaceOpt) *
 func testRequireEqualFP(t *testing.T, fp, expected *FlightPlan) {
 	t.Helper()
 
-	require.Len(t, expected.Modules, len(fp.Modules))
-	require.Len(t, expected.ScenarioBlocks, len(fp.ScenarioBlocks))
-	require.Len(t, expected.Providers, len(fp.Providers))
+	require.Len(t, fp.Modules, len(expected.Modules))
```

Comment:
These were in the wrong order which I discovered while testing my in-flight changes, so I fixed them here.